### PR TITLE
Remove `allotedSeat` and `allotedRoom` from Checkin 

### LINKED ISSUE #2002: [BUG] CheckIn doesn't work
**Describe the bug**
The CheckIn feature for events doesn't work

**To Reproduce**

1. Make an event
2. Add some attendees
3. Login with Admin/SuperAdmin
4. Try to goto Event Dashboard and checkIn

**Expected behavior**
The CheckIn option should list all users and allow us to checkIn

**Actual behavior**
Throws error

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

Related issue: PalisadoesFoundation/talawa-admin#1722

### PR DESCRIPTION
**Pull request by lakshz**
**What kind of change does this PR introduce?**
This PR removes any instance of `allotedRoom` and `allotedSeat` in the codebase

**Issue Number:**

Fixes #2002 

**Did you add tests for your changes?**
Y

**Snapshots/Videos:**
N/A

**If relevant, did you update the documentation?**

**Summary**

**Does this PR introduce a breaking change?**

**Other information**

**Have you read the [contributing guide](https://github.com/PalisadoesFoundation/talawa-api/blob/master/CONTRIBUTING.md)?**

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -66,6 +66,9 @@ jobs:
             **.ts
             '!src/types/**.ts'
 
+      - name: Echo changed files
+        run: echo "${{ env.CHANGED_FILES }}"
```
Comment (palisadoes) [inline .github/workflows/pull-request.yml:70]: Revert all changes to this file. It is out of scope of the PR

**Review by palisadoes — CHANGES_REQUESTED** [review]
See comments

**Comment by lakshz** [comment]
@palisadoes Lint tests were failing due to an error. I'm trying to resolve that error, the reason I echoed changed-files in the workflow

**Comment by lakshz** [comment]
I see something is wrong with the `changed-files` actions. It is returning wrong files changed in the PR.

Let me work on fixing this.

**Comment by lakshz** [comment]
Fixed after updating the version for `tj-actions/changed-files`.  Probably, the error was resolved in v44 recently:

Screenshot 2024-04-06 at 19 55 28

**Review by palisadoes — CHANGES_REQUESTED** [review]
Revert all changes to `pull-request.yml` 

It is out of scope of the issue

**Comment by lakshz** [comment]
@palisadoes Changes to `pull-request.yml` were done to make the tests pass.

**Review by palisadoes — APPROVED** [review]

**Pull request merged**